distributed/protocol/compression.py

```diff
@@ -10,6 +10,13 @@
 import dask
 from tlz import identity
 
+
+class InvalidCompressionMethodError(ValueError):
+    def __init__(self, method):
+        self.method = method
+        super().__init__("Invalid compression method: %s" % method)
```

Comment:
We have historically not created custom error classes within the Dask projects, and have instead tended towards the base Value/Type/ImportErrors.  This isn't a rule by any means, but making a type for this is a bit odd for the project.

Reply:
:+1: thanks for the feedback! Simplified this to a `ValueError` in https://github.com/dask/distributed/pull/3742/commits/d712c1d699436f1ae6b8bbcab52b2cf0993561ce